apps/docs/content/guides/database/prisma.mdx

```diff
@@ -0,0 +1,448 @@
+      ```
+    </StepHikeCompact.Code>
+
+  </StepHikeCompact.Step>
+
+    <StepHikeCompact.Step step={2}>
+        <StepHikeCompact.Details title="Create a Prisma Project">
+        Create a new prisma Project on your computer
+        </StepHikeCompact.Details>
+
+        <StepHikeCompact.Code>
+
+            Create a new directory
+            ```bash Terminal
+            mkdir hello-prisma
+            cd hello-prisma
+            ```
+
+            Initiate a new Prisma project
+          <Tabs
+            scrollable
+            size="small"
+            type="underlined"
+            defaultActiveId="npm_initiate"
+            queryGroup="initiate"
+          >
+            <TabPanel id="npm_initiate" label="npm">
+              ```bash
+              npm init -y
+              npm install prisma typescript ts-node @types/node --save-dev
+
+              npx tsc --init
+
+              npx prisma init
+              ```
+            </TabPanel>
+            <TabPanel id="pnpm_initiate" label="pnpm">
+              ```bash
+              pnpm init -y
+              pnpm install prisma typescript ts-node @types/node --save-dev
+
+              pnpx tsc --init
+
+              pnpx prisma init
+              ```
+            </TabPanel>
+            <TabPanel id="yarn_initiate" label="yarn">
+              ```bash
+              yarn init -y
+              yarn install prisma typescript ts-node @types/node --save-dev
+
+              yarn dlx tsc --init
+
+              yarn dlx prisma init
+              ```
+            </TabPanel>
+            <TabPanel id="bun_initiate" label="bun">
+              ```bash
+              bun init -y
+              bun install prisma typescript ts-node @types/node --save-dev
+
+              bunx tsc --init
+
+              bunx prisma init
+              ```
+            </TabPanel>
+          </Tabs>
+        </StepHikeCompact.Code>
+
+  </StepHikeCompact.Step>
+
+  <StepHikeCompact.Step step={3}>
+    <StepHikeCompact.Details title="Add your connection information to your .env file">
+    - Visit the [Database Settings](https://supabase.com/dashboard/project/_/settings/database)
+    ![Session Mode](/docs/img/database/session_mode.png)
+    - Find your Supavisor Session Mode string. It should end with 5432. It will be used in your `.env` file.
+    <Admonition type="note">
+      If you're in an [IPv6 environment](https://github.com/orgs/supabase/discussions/27034) or have the IPv4 Add-On, you can use the direct connection string instead of Supavisor in Session mode.
+    </Admonition>
+
+    - If you plan on deploying Prisma to a serverless or auto-scaling environment, you'll also need your Supavisor transaction mode string.
+    ![Session Mode](/docs/img/guides/database/transaction-mode.png)
+    - The string is identical to the session mode string but uses port 6543 at the end.
+
+
+    </StepHikeCompact.Details>
+
+    <StepHikeCompact.Code>
+            <Tabs>
```

Comment:
```suggestion
            <Tabs scrollable size="small" type="underlined">
```

should probably make this the default setting, but for now usage is documented here: https://supabase.com/docs/contributing?queryGroups=language&language=js#tabs